File: benchmarks/javascript-membership-index/baseline/workload.js
const items = Array.from({ length: 12000 }, (_, index) => index);
const queries = Array.from({ length: 12000 }, (_, index) => index + 6000);
const result = queries.reduce((count, query) => count + Number(items.includes(query)), 0);
if (result !== 6000) throw new Error(`unexpected result: ${result}`);

File: benchmarks/invariant-hoist/workload.py
def rank_queries(values, queries):
    result = []
    for query in queries:
        ordered = sorted(values)
        result.append((query, ordered[0], ordered[-1]))
    return result


values = list(range(6000, 0, -1))
queries = list(range(250))
rank_queries(values, queries)
